### ISSUE TITLE
Unable to test this function due to inaccessibility of the external function.

### ISSUE THREAD

**Issue by ryu9827** [tikonoff/staking-app#5]
https://github.com/tikonoff/staking-app/blob/8306bb8deebe0f554e99bf4e525ea4a3f5672397/contracts/HumanStandardToken.sol#L54

This line of codes call "receiveApproval()" which is not included in this project/repo. 
I assume the contract and the function are existing somewhere, but they are not under auditing this time.
And according to [Solidity 0.4.20 documentation](http://solidity.readthedocs.io/en/develop/types.html?highlight=keccak), `bytes32` is redundant.
